spec/components/email/styles_spec.rb

```diff
@@ -326,7 +326,7 @@ def strip_and_inline
         strip_and_inline
         expect(@frag.to_s).to include("cid:email/test.png")
         expect(@frag.to_s).to include("cid:email/test2.ico")
-        expect(@frag.css('[data-sripped-secure-media]')).not_to be_present
```

Comment:
https://github.com/discourse/discourse/pull/12779#discussion_r617142908
What is this test trying to prove? (👍)

https://github.com/discourse/discourse/pull/12779#discussion_r618720707
@riking Change looks good, `data-stripped-secure-media` refers to "image removed, click to view" messages

https://github.com/discourse/discourse/pull/12779#discussion_r618809689
⚠️ n.b. this probably means the test was accidentally passing / the code wasn't being tested.